core-bundle/src/Resources/contao/widgets/ModuleWizard.php

```diff
@@ -75,7 +75,7 @@ public function generate()
 		// Add the module type (see #3835)
 		foreach ($modules as $k=>$v)
 		{
-			$v['type'] = $GLOBALS['TL_LANG']['FMD'][$v['type']][0];
+			$v['type'] = $GLOBALS['TL_LANG']['FMD'][$v['type']][0] ?: $v['type'];
```

Comment:
```suggestion
			$v['type'] = $GLOBALS['TL_LANG']['FMD'][$v['type']][0] ?? $v['type'];
```

Otherwise https://3v4l.org/PPYtg

Reply:
Ah, that does not work in Contao 4.4, because we still support PHP 5.6. 🤷🏻‍♂️